server/scripts/psql-migration-test.sh

```
@@ -14,15 +14,15 @@ docker exec -i mattermost-postgres psql -U mmuser -d migrated -c "INSERT INTO Sy
 
 echo "Setting up config for db migration"
 cat config/config.json | \
-    jq '.SqlSettings.DataSource = "postgres://mmuser:mostest@localhost:5432/migrated?sslmode=disable&connect_timeout=10"'| \
+    jq '.SqlSettings.DataSource = "postgres://mmuser:mostest_password@localhost:5432/migrated?sslmode=disable&connect_timeout=10"'| \
     jq '.SqlSettings.DriverName = "postgres"' > $TMPDIR/config.json
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)